### ISSUE TITLE
Feature Request: Export the number of RTR client connections via the /metrics endpoint

### ISSUE THREAD

**Issue by tardoe** [NLnetLabs/routinator#334]
To be able to better monitor RTR connectivity to clients, it would be ideal to export a Prometheus gauge metric that indicate the number and status of connected RTR clients.

Something along the lines of this would suffice:
```
# HELP routinator_rtr_clients number of connected RTR client
# TYPE routinator_rtr_clients gauge
routinator_rtr_clients 2

# HELP routinator_rtr_client_status status of RT clients, 1 = connected
# TYPE routinator_rtr_client_status gauge
routinator_rsync_status{client="192.2.0.1"} 1
routinator_rsync_status{client="192.2.0.2"} 1
```

**Comment by partim** [comment]
Some information for RTR is already available in 0.7.0:

```
# HELP routinator_rtr_connections total number of RTR connections
# TYPE routinator_rtr_connections counter
routinator_rtr_connections 0

        
# HELP routinator_rtr_current_connections currently open RTR connections
# TYPE routinator_rtr_current_connections gauge
routinator_rtr_current_connections 0

        
# HELP routinator_rtr_bytes_read number of bytes read via RTR
# TYPE routinator_rtr_bytes_read counter
routinator_rtr_bytes_read 0

        
# HELP routinator_rtr_bytes_written number of bytes written via RTR
# TYPE routinator_rtr_bytes_written counter
routinator_rtr_bytes_written 0
```

The list and status for individual clients sounds like a good idea, indeed. We’ll look into adding that in one of the next versions.

**Comment by partim** [comment]
For the per-connection statistics, we want to also add time since they were last updated. This, however, needs a bit more refactoring. In order to not delay 0.8 any further and since we have more fine-grained metrics planned, we will move the RTR per-connection metrics to 0.9.
